docs/config.ld

```diff
@@ -21,6 +21,7 @@ topics={
     '02-contributing.md',
     '03-declarative-layout.md',
     '04-new-widgets.md',
+    '99-FAQ.md',
```

Comment:
Sure, I can do that. However, if we need to add something later we can just rename this file then. No idea what a good order would actually be...